app/lib/analyzer/backend.dart

```diff
@@ -275,21 +275,26 @@ class AnalysisBackend {
   /// - it is **not** the latest for the pana version, or
   /// - it is older than 6 months (except if it is the latest one).
   Future deleteObsoleteAnalysis(String package, String version) async {
+    final pvaKey = db.emptyKey
+        .append(PackageAnalysis, id: package)
+        .append(PackageVersionAnalysis, id: version);
+    final PackageVersionAnalysis pva = (await db.lookup([pvaKey])).single;
```

Comment:
`.single` will throw an exception if there's not exactly one element in the list. I don't think that's what we want here.

Reply:
The `db.lookup` will return with a List the same size as the input list, and `.single` is just as good as `.first` or `[0]`. I can change it to either of them.

Reply:
In that case, `.single` should be fine. Thanks!